docs/content/migration.md

```diff
@@ -0,0 +1,542 @@
+| sidenav-focus      |   [Docs][95] |   -          | [autofocus][96] only supports focusing the first focusable element.|
+| switch             |   [Docs][61] |   [Docs][14] | `mat-slide-toggle`                                     |
+| slider             |   [Docs][62] |   [Docs][16] |                                                        |
+| slider-container   |   [Docs][97] |   -          | Not implemented                                        |
+| subheader          |   [Docs][98] |   [Docs][99] | `matSubheader`                                         |
+| swipe              |   [Docs][100]|   -          | See [HammerJS setup][101] and [Hammer.Swipe][102]      |
+| tabs               |   [Docs][64] |   [Docs][13] |                                                        |
+| textarea           |   [Docs][52] |   [Docs][5]  |                                                        |
+| toolbar            |   [Docs][66] |   [Docs][7]  |                                                        |
+| tooltip            |   [Docs][67] |   [Docs][18] |                                                        |
+| truncate           |   [Docs][103]|   -          | Not implemented                                        |
+| virtual-repeat     |   [Docs][68] |   [Docs][40] | `cdk-virtual-scroll-viewport` and `*cdkVirtualFor`     |
+| whiteframe         |   [Docs][104]|  [Guide][105]| Based on classes and mixins                            |
+
+### Services
+
+| Service          | Old          | New          | Special Notes                                          |
+|------------------|--------------|--------------|--------------------------------------------------------|
+| $mdAriaProvider  |   [Docs][124]|   -          | Not implemented                                        |
+| $mdBottomSheet   |   [Docs][42] |   [Docs][38] | `MatBottomSheet`                                       |
+| $mdColors        |   [Docs][125]|   -          | [`mat-color` mixin][74] supports static theme color lookups.|
+| $mdCompiler      |   [Docs][126]|   -          | Not implemented                                        |
+| $mdCompilerProvider| [Docs][127]|   -          | Not implemented                                        |
+| $mdDateLocaleProvider|[Docs][128]|  [Docs][129]| `MomentDateAdapter`                                    |
+| $mdDialog        |   [Docs][48] |   [Docs][22] | `MatDialog`                                            |
+| $mdGestureProvider|  [Docs][130]|   -          | Not implemented                                        |
+| $mdIcon          |   [Docs][131]|   [Docs][132]| `svgIcon`                                              |
+| $mdIconProvider  |   [Docs][133]|   [Docs][134]| `MatIconRegistry`                                      |
+| $mdInkRipple     |   [Docs][58] |   [Docs][19] | `matRippleTrigger`                                     |
```

Comment:
Is it right to mention the `matRippleTrigger` input here? Sounds a bit confusing to me. How about referring to the `MatRipple` directive instead and linking to the guide?

e.g. https://material.angular.io/components/ripple/overview#manual-ripples

Reply:
The thought was that `$mdInkRipple` attaches ripples to an element and I had thought that `matRippleTrigger` was the way to do that. Re-reading the docs, it looks like the `matRippleTrigger` reference should be removed. Thanks.